envoy/router/shadow_writer.h

```diff
@@ -34,7 +34,8 @@ class ShadowWriter {
    * @param cluster supplies the cluster name to shadow to.
    * @param headers supplies the headers for initializing the shadow.
    * @param options supplies the request options for the underlying asynchronous request.
-   * @return OngoingRequest* pointer which can have additional data and trailers sent to it.
+   * @return OngoingRequest* pointer owned by the AsyncClient which can have additional data and
```

Comment:
fair point on consistency re pointer.  can it be null?  let' comment either way.